src/bioregistry/data/bioregistry.json

```diff
@@ -91098,7 +91098,7 @@
         "description": "Convert SMILES to various molecular formats using different toolkits.",
         "homepage": "https://api.naturalproducts.net/latest/docs#/convert/SMILES_convert_to_Formats_convert_formats_get",
         "name": "Convert SMILES to various molecular formats via the Cheminformatics Microservice",
-        "uri_format": "https://api.naturalproducts.net/latest/convert/formats?smiles=$1&toolkit=cdk"
+        "uri_format": "https://api.naturalproducts.net/latest/convert/formats?smiles=$1&toolkit=$2"
```

Comment:
The Bioregistry doesn't currently support this kind of syntax with multiple placeholders, but this is something we could consider in the future if you make an issue about it